docs/src/PolyhedralGeometry/polyhedral_complexes.md

```diff
@@ -0,0 +1,54 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["polyhedral_complexes.md"]
+```
+
+# Polyhedral Complexes
+
+## Introduction
+
+Let $\mathbb{F}$ be an ordered field; the default is that
+$\mathbb{F}=\mathbb{Q}$ is the field of rational numbers and other fields are
+not yet supported everywhere in the implementation.
+
+A nonempty finite collection $\mathcal{P}$ of polyhedra in
+$\mathbb{F}^n$, for $n$ fixed, is a *polyhedral complex* if
+
+- the set $\mathcal{F}$ is closed with respect to taking faces and
+- if $C,D\in\mathcal{F}$ then $C\cap D$ is a face of both, $C$ and $D$.
```

Comment:
I am unsure why I am being questioned in this way. I marked both this and #889 as drafts since they are not finished. You are of course free to give feedback, but you might remark on things that we intend to change anyway. I thought marking a PR as _draft_ implies that feedback at this point is not needed yet.

Reply:
Hi Lars, sorry, only saw your comment here just now. To be clear: I did not mean to "question" you (like an inquisitor), to the contrary: what I meant was this: I saw the one PR and had some immediate feedback. Then I saw this PR. Then I noticed that they are marked as draft, and now was like "oh, ok, maybe he doesn't even want feedback?!?".

My apologies for very badly communicating this!